### DISCUSSION TITLE
looking for a partner to maintain / develop my online shop

### DISCUSSION THREAD

**Discussion by Fred-youmi** [OpenMage/magento-lts#4174] [Hire]
Hello everyone,
I run a small online shop based on Openmage (OpenMage 20.1.0), which opened in 2021, and is based in France.
This CMS is associated with the ERP DOLIBARR.
I personally manage the site's operations, marketing and sales.
The person who managed the technical aspects (initialisation/maintenance/development) is now working on other projects and can no longer manage this one.
So I'm looking for someone to take over.
The idea is to:
- maintain the functionality (support)
- add others: for example, integrating the Alma payment in instalments module; modifying the modes of transport offered...
Someone who also knows Dolibarr would be ideal.
But, failing that, I'm thinking of working with another person for the Dolibarr ERP part.
In that case, we'll need to co-ordinate properly in the event of any changes.
Broadly speaking, the site's working well (thanks openmage!), but what I really want is to have someone to talk to in case of problems, and who's capable of having a vision to support the site's development over the medium to long term.
I'm open to working with a freelancer or an agency.
More details in private.

**Comment by hiepnt93** [comment]
please contact me , i'm open for working
here are my linked in https://www.linkedin.com/in/hiep-nguyen-hd/

**Comment by fballiano** [comment]
@Fred-youmit former maintainer here, feed free to contact me at https://fabrizioballiano.com

**Comment by llvee** [comment]
Up to help if you are still in need of assistance. I have performed some development work using every popular ecommerce system & first used PHP many years ago. I am familiar with PHP best practices, engineering complex applications & services using PHP.

**Comment by sreichel** [comment]
I can help too.

Please close if solved,

**Reply by Fred-youmi** [reply]
Thank you for your message; Indeed I have already found my Magento support partner. I'll keep your contact details in case

**Comment by commercebro** [comment]
Hello @Fred-youmi If you need any suggestions/recommendations regarding openmage or support in website maintenance and development - feel free to contact me. I have an experience in magento 1 support, openmage, Openmage->M2 migration. If you need any recommendations - I'll be able to provide.

**Comment by Fred-youmi** [comment]
H‌ello Uladzimir ,

Thank you very much for your message.
But I have already someone to do this.
However I'll keep your email just in case.

Regards,

Frédéric JEANNIN


- Gérant -

+33(0)6 84 57 40 34 


LGF Distribution - Let's Go Fishin'!

357 chemin du mas de Bouisson - 34680 St Georges d'Orques
www.seabasslureshop.com 


 

De : "Uladzimir Plakhotnikau" ***@***.***>
A : "OpenMage/magento-lts" ***@***.***>,"Fred-youmi" ***@***.***>,"Mention" ***@***.***>
Envoyé: lundi 21 Avril 2025 17:11
Objet : Re: [OpenMage/magento-lts] looking for a partner to maintain / develop my online shop (Discussion #4174)
 

 

Hello @Fred-youmi If you need any suggestions/recommendations regarding openmage or support in website maintenance and development - feel free to contact me. I have an experience in magento 1 support, openmage, Openmage->M2 migration. If you need any recommendations - I'll be able to provide.
